trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/import_airfoil_geometry.py

```diff
@@ -68,15 +68,17 @@ def  import_airfoil_geometry(airfoil_geometry_files, npoints = 100):
 
         # Check if it's a Selig or Lednicer file
         format_line = f.readline()
```

Comment:
Update the header with your name/date modified

Reply:
Addressed